framework/auth/cas.py

```diff
@@ -0,0 +1,89 @@
+import furl
+import json
+import requests
+
+from lxml import etree
+
+
+class CasResponse(object):
+
+    authenticated = False
```

Comment:
These should probably be instance variables rather than class variables. That way, setting them on once instance doesn't affect other instances.